### PR TITLE
Add Landing Page Unit tests

### PR DESCRIPTION
**Pull request by jeff-phillips-18**

### CODE REVIEW

File: dist/origin-web-catalogs.css [hunk 1]
```diff
@@ -59,7 +59,7 @@
   position: fixed;
   right: 320px;
   top: 60px;
-  z-index: calc(1029);
+  z-index: calc(980);
```
Comment (dtaylor113) [inline dist/origin-web-catalogs.css]: What does this need to be calc'ed?

File: test/landing-page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  beforeEach(() => {
+    inject(function (_$window_: any, _$timeout_: any) {
+      $window = _$window_;
+      $timeout = _$timeout_;
+    });
+  });
+
+  beforeEach(() => {
+    var landingPageHtml: string = '' +
+      '<landing-page search-placeholder="Test Placeholder">' +
```
Comment (dtaylor113) [inline test/landing-page.spec.ts]: Is '' + '...' really necessary?

File: dist/origin-web-catalogs.css (hunk 1 above)
Comment (jeff-phillips-18) [inline dist/origin-web-catalogs.css]: It is calc'd by:

z-index: calc(@zindex-navbar-fixed - 50);

in the less file. Idea is to keep it under the navbar but over the main panel.

Comment (dtaylor113) [inline dist/origin-web-catalogs.css]: ah, right, this is a 'dist' file

File: test/landing-page.spec.ts (hunk 2 above)
Comment (jeff-phillips-18) [inline test/landing-page.spec.ts]: It is building up the html string. I find it easier to understand if it is on multiple lines.

File: dist/origin-web-catalogs.css (hunk 1 above)
Comment (spadgett) [inline dist/origin-web-catalogs.css]: @jeff-phillips-18 Can't this just be?

```less
z-index: @zindex-navbar-fixed - 50;
```

**Pull request merged**